services/manage.py

```diff
@@ -1,22 +1,35 @@
 import webapp2
 import json
+import datetime
+import logging
+
 
 from models.stream import Stream
 
 class Manage(webapp2.RequestHandler):
 
     def get(self):
         self.response.headers['Content-Type'] = 'application/json'
-        streams_query = Stream.query()
-        streams = streams_query.fetch()
+        user = self.request.get('user')
+        stream_type = self.request.get('type')
+        if stream_type == 'own' :
+            streams = Stream.query(Stream.owner == user)
+        else:
+            #TODO : handle suscription views
+            streams = [] 
+        
         stream_list = []
 
-        for stream in streams:
-            stream_list.append({
-            'id': stream.key.id(), 
-            'name': stream.name, 
-            'cover_url': stream.cover_url
-            })
+        if streams is not None:
```

Comment:
No need for the if I think.

Reply:
it errors out privately in the console but it doesn't affect the frontend, it was more for me to control the console errors :)